src/shared/views/LoadMore.js

```diff
@@ -117,29 +117,27 @@ class _LoadMoreCard extends React.Component<Props> {
     };
 
     render() {
-        const { tutorial } = this.props;
+        const { t, tutorial } = this.props;
         return (
             <View style={styles.congratulationsSlide}>
                 <Text style={styles.finishedText}>
-                    {tutorial
-                        ? 'Good. You have completed the tutorial. You are ready to do some mapping!'
-                        : 'Great job! You finished this group.'}
+                    {tutorial ? t('completedTutorial') : t('finishedGroup')}
                 </Text>
 
                 <Button
                     style={styles.moreButton}
                     onPress={this.onComplete}
                     textStyle={{ fontSize: 18, color: COLOR_WHITE }}
                 >
-                    {tutorial ? "Let's go!" : 'Complete Session'}
+                    {tutorial ? t('letsGo') : t('completeSession')}
```

Comment:
this is the way of writing?
```
if tutorial then "let's go"
else "complete"
```

Reply:
Yep, it's a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) which the react docs recommend as one way to do conditional rendering.